alacritty_terminal/Cargo.toml

```diff
@@ -29,7 +29,8 @@ dirs = "3.0.1"
 
 [target.'cfg(unix)'.dependencies]
 nix = "0.22.0"
-signal-hook = { version = "0.1", features = ["mio-support"] }
+signal-hook = "0.3"
+signal-hook-mio = { version = "0.2", features = ["support-v0_6"] }
```

Comment:
Please specify the full latest versions (including fixup) of these crates unless you've explicitly verified that 0.3.0 and 0.2.0 work too (in which case this should be signified by explicitly specifying that.